docs/manual/java/gettingstarted/index.toc

```diff
@@ -2,10 +2,8 @@ IntroGetStarted:Introduction
 JavaPrereqs:Lagom Java prerequisites;next=GettingStartedMaven,GettingStartedSbt
 GettingStartedMaven:Creating and running Hello World with Maven;next=UnderstandHello
 GettingStartedSbt:Creating and running Hello World with sbt;next=UnderstandHello
-UnderstandHello:Understanding Hello World;next=MavenIDEs,EclipseSbt,IntellijSbt
-MavenIDEs:Integrating Maven projects with an IDE;next=LagomExamples
-EclipseSbt:Integrating sbt with Eclipse;next=LagomExamples
-IntellijSbt:Integrating sbt with IntelliJ;next=LagomExamples
+UnderstandHello:Understanding Hello World
 LagomExamples:Lagom examples
+WhatsNext:What's Next?
```

Comment:
But I agree that we should be consistent, and if there is an American standard for headings, we should adopt that rather than the Australian standard. If there is no standard, then my vote would be for sentence case, simply because I find it easier to read.